packages/integration-tests/src/components/accessibility/test-delegate-focus-click-input-in-negative-tabindex-previous-sibling-focused/delegate-focus-click-input-in-negative-tabindex-previous-sibling-focused.spec.js

```diff
@@ -10,34 +10,18 @@ const assert = require('assert');
 // the previous sibling to the custom element
 
 describe('Delegates focus', () => {
-    const URL = '/delegate-focus-click-input-in-negative-tabindex-previous-sibling-focused';
```

Comment:
Please add a line in the README(integration-tests/README.md) how to get the url for each suite for anybody not familiar with this setup 🙏